test/tests/divertor_monoblock/tests

```diff
@@ -0,0 +1,21 @@
+[Tests]
+  issues = '#144'
+  design = 'divertor_monoblock/index.md'
+  [Shimada2024_input_check]
+    type = RunApp
+    input = 'divertor_monoblock.i'
+    requirement = 'The system shall maintain a working input file to model heat and tritium transport in a divertor monoblock during pulsed operation.'
+    check_input = True
+    method = opt
+  []
+  [Shimada2024_run]
```

Comment:
Out of curiosity, how long does this take to run?

Reply:
Around 1 minute with the coarser mesh and smaller simulation time. So very reasonable for a heavy test, ut definitely a bit much for a regular test.